### DISCUSSION TITLE
Runner do not pick up jobs

### DISCUSSION THREAD

**Discussion by fryorcraken** [community/community#32262] [Actions]
For the past ~24 hours, runners have been very slow at picking up jobs.

e.g.: 

- https://github.com/status-im/js-waku/runs/8240338968?check_suite_focus=true (25min, still not picked up)
- https://github.com/status-im/js-waku/runs/8240219393?check_suite_focus=true (40min, still not picked up)
- https://github.com/status-im/js-waku/actions/runs/3007186525 took more than 1h20min to be picked up

But no incident is reported in GitHub Status.

**Comment by Rexpert** [comment]
The GitHub scheduled runner is expected to be delayed in most of the time. You may use an external scheduler (for eg: [https://cron-job.org/](https://cron-job.org/)) to set trigger your action. 

Ref: [https://github.com/orgs/community/discussions/27130](https://github.com/orgs/community/discussions/27130)
